Gaijinizer patch/Enemies.txt

```diff
@@ -2441,11 +2441,12 @@ No one survived her fierce footsteps.#MTLed
 これは、古来の妖狐がその実力を誇示するために
 毛並みや尾数をステイタスにしていた名残だという。
 > CONTEXT: Enemies/599/BookDescription/3
-Not only the dakki, but the tribe of sorceress
-Always cherish your own tail and ear coat.
-This is for the ancient sorcery to show off its ability
-It is said that it is a remnant that the coat and the
-number of tails were sit tass.#MTLed
+To all Kitsune, Daji included, one's tail and
+fur is something to always be diligently
+maintained. The reason for this is said to be
+that ancient Kitsune used the number of tails as
+well as the condition of their coats as a display
+of their status and prowess.#RonIffyWording
 > END STRING
```

Comment:
Some reworking for flow:
>_**To all Kitsune, Daji included, one's tails and
fur are always something that is diligently
maintained. The reason behind this is that the
quality of fur and number of tails was a display of
status and strength since ancient times for them.**_